flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/aggregate/AggregateUtil.scala

```diff
@@ -916,6 +916,10 @@ object AggregateUtil {
                   new DecimalMinAggFunction
                 case BOOLEAN =>
                   new BooleanMinAggFunction
+                case VARCHAR =>
+                  new StringMinAggFunction
+                case CHAR =>
```

Comment:
The char type occurs if you use literals in SQL e.g. "SELECT 'hello' " but the runtime should actually treat char and varchar the same as they are both strings.